src/App.css

```diff
@@ -0,0 +1,10 @@
+.app {
+    display: flex;
+  }
+  
+  .main-content {
+    margin-left: 60px; /* Same width as the sidebar */
+    padding: 20px;
+    width: calc(100% - 60px);
```

Comment:
Why this? Why arent we using margins of 60px

Reply:
Using a margin of 60px on all sides of .main-content would push it away from surrounding elements, potentially causing layout issues and misalignment. By specifically setting the left margin to 60px, the main content area is shifted away from the sidebar while keeping the rest of its layout unaffected.
The calc(100% - 60px) width calculation ensures that the main content area takes up the remaining width after subtracting the width of the sidebar, making it responsive to changes in screen size or layout adjustments.

Reply:
I will be working on disabling mobile view of the application after this.